python/pyspark/pandas/data_type_ops/base.py

```diff
@@ -39,9 +41,8 @@
     TimestampType,
     UserDefinedType,
 )
-
 import pyspark.sql.types as types
```

Comment:
I'm wondering why we import some types from `pyspark.sql.types` but also import `types` here?

Reply:
Makes sense, adjusted.